java/README.md

```diff
@@ -0,0 +1,67 @@
+# Java humbug library
+
+## How to use:
+### Install
+* Download source code
+* Add `humbug/` folder from `src/humbug/ ` to your project
+* Copy dependencies from [dependencies](#dependencies) and add to your exsisting 
```

Comment:
@kompotkot : We should set up a CI process that releases this to Maven central.

We can follow this documentation: https://docs.github.com/en/actions/guides/publishing-java-packages-with-maven

Reply:
We decided to use GitHub packages for now - will revisit this decision if they start charging us.